roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -201,8 +201,8 @@
     - name: Pull and tag certsuite image for the version under test
       ansible.builtin.shell: |
         set -ex
-        podman pull quay.io/testnetworkfunction/{{ kbpc_image_name }}:{{ kbpc_version_image }}
-        podman tag quay.io/testnetworkfunction/{{ kbpc_image_name }}:{{ kbpc_version_image }} \
+        podman pull quay.io/redhat-best-practices-for-k8s/{{ kbpc_image_name }}:{{ kbpc_version_image }}
+        podman tag quay.io/redhat-best-practices-for-k8s/{{ kbpc_image_name }}:{{ kbpc_version_image }} \
```

Comment:
Not really sure about a current parity between the old repo and the new one, seems like there are some images that are still missing:

```Shell
## Legacy repo
❯ skopeo inspect docker://quay.io/testnetworkfunction/cnf-certification-test:v5.2.3 | jq .Name
"quay.io/testnetworkfunction/cnf-certification-test"

## New repo
❯ skopeo inspect docker://quay.io/redhat-best-practices-for-k8s/certsuite:v5.2.3 | jq .Name
FATA[0000] Error parsing image name "docker://quay.io/redhat-best-practices-for-k8s/certsuite:v5.2.3": reading manifest v5.2.3 in quay.io/redhat-best-practices-for-k8s/certsuite-probe: manifest unknown 

```

Reply:
The image jobs have ran: https://quay.io/repository/redhat-best-practices-for-k8s/certsuite?tab=tags

Reply:
Good, I see the expected tag version here:

```Shell
$ crane ls quay.io/redhat-best-practices-for-k8s/certsuite
latest
unstable
v5.2.3
```